lua/entities/gmod_tardis_interior/modules/cl_customlighting_prompt.lua

```diff
@@ -0,0 +1,29 @@
+--Require Lighting Override
```

Comment:
I think since this is similar enough to the other one we can actually put them both into the same file. GMod has limits on how many Lua files can be loaded so I like to reduce them when possible. Can we merge the two files together into like `cl_prompts.lua`?